notifications/notifications.py

```diff
@@ -40,12 +40,12 @@ def get_enabled_handlers(self):
             enabled_handlers.append("Pavlok")
         return enabled_handlers
 
-    def send_notification(self, message):
+    def send_notification(self, message, options={}):
         with ThreadPoolExecutor(
             max_workers=len(self.get_enabled_handlers())
         ) as executor:
             if self.audio_handler.enabled:
-                executor.submit(self.audio_handler.play)
+                executor.submit(self.audio_handler.play, options.get("audio"))
```

Comment:
The way I've gone about this feels pretty poor.. Would appreciate feedback/suggestions on a more Pythonic solution.

The general idea here is that it would be useful for specific notification handlers to be able to receive their own set of options. The structure and supported options would differ between NotificationHandler types.

Reply:
It's a bit confusing at first sight, but it makes sense and undoubtedly gets the job done.